### PR TITLE
Fix build issues

### PR DESCRIPTION
**Pull request by Chraser**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,7 +5,7 @@
   <meta charset="UTF-8" />
   <link href="https://fonts.googleapis.com/css2?family=Material+Icons" rel="stylesheet">
   <link rel="icon" type="image/svg+xml" href="/src/favicon.svg" />
-  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="/style.css">
```
Comment (Techypanda) [inline index.html:8]: this shouldn't matter should it? (I thought html translated style.css -> /style.css on the root domain)

File: tsconfig.json [hunk 2]
```diff
@@ -4,7 +4,7 @@
     "useDefineForClassFields": true,
     "lib": ["DOM", "DOM.Iterable", "ESNext"],
     "allowJs": false,
-    "skipLibCheck": false,
+    "skipLibCheck": true,
```
Comment (Techypanda) [inline tsconfig.json:7]: 'TypeScript will type check the code you specifically refer to in your app’s source code.' are we sure we want this?

Comment (Techypanda) [reply tsconfig.json:7]: resolved in dm

**Review by kevinle-1 — CHANGES_REQUESTED** [review]
Don't think we should enable skip library checks, see https://stackoverflow.com/a/56268670/11378789

File: index.html (hunk 1 above)
Comment (Techypanda) [reply index.html:8]: resolved in dm

**Review by Techypanda — APPROVED** [review]

**Review by kevinle-1 — APPROVED** [review]

**Comment by kevinle-1** [comment]
Not a massive issue 8)

**Pull request merged**